data/WI/incentives.json

```diff
@@ -0,0 +1,667 @@
+    "authority_type": "state",
+    "authority": "wi-focus-on-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "wi_focusOnEnergy_insulation&AirSealingRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 75,
+      "minimum": 75,
+      "maximum": 75
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$75 for Duct Sealing & Insulation."
+    },
+    "start_date": 2023,
+    "end_date": 2023,
+    "low_income": "wi-focus-on-energy"
+  },
+  {
+    "id": "WI-6",
+    "authority_type": "state",
+    "authority": "wi-focus-on-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "wi_focusOnEnergy_insulation&AirSealingRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 200,
+      "minimum": 200,
+      "maximum": 200
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$200 for DIY Attic Insulation & Air Sealing."
+    },
+    "start_date": 2023,
+    "end_date": 2023,
+    "low_income": "wi-focus-on-energy"
+  },
+  {
+    "id": "WI-7",
+    "authority_type": "state",
+    "authority": "wi-focus-on-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "smart_thermostat",
+    "program": "wi_focusOnEnergy_smartThermostatsRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 50,
+      "minimum": 50,
+      "maximum": 50
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$50 for Smart Thermostats."
+    },
+    "start_date": 2023,
+    "end_date": 2023,
+    "bonus_available": true,
+    "low_income": "wi-focus-on-energy"
+  },
+  {
+    "id": "WI-8",
+    "authority_type": "state",
+    "authority": "wi-focus-on-energy",
+    "payment_methods": [
+      "unknown"
+    ],
+    "item": "heat_pump_water_heater",
+    "program": "wi_focusOnEnergy_waterHeatingRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 300,
+      "minimum": 300
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Instant discounts for electric heat pump water heaters starting at $300 through a Trade Ally contractor."
+    },
+    "bonus_available": true,
+    "low_income": "wi-focus-on-energy"
+  },
+  {
+    "id": "WI-9",
+    "authority_type": "state",
+    "authority": "wi-focus-on-energy",
+    "payment_methods": [
+      "unknown"
+    ],
+    "item": "other",
+    "program": "wi_focusOnEnergy_waterHeatingRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 120,
+      "minimum": 120
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Instant discounts for natural gas tankless water heaters start at $120 through a Trade Ally contractor."
```

Comment:
We absolutely do not want to show incentives for gas appliances :)

You can delete this from the source spreadsheet. Same for W12-W20.

Reply:
Good catch, thank you!